packages/app-store/stripepayment/lib/PaymentService.ts

```diff
@@ -61,7 +61,6 @@ export class PaymentService implements IAbstractPaymentService {
         amount: payment.amount,
         currency: this.credentials.default_currency,
         payment_method_types: ["card"],
-        application_fee_amount: paymentFee,
       };
```

Comment:
Verified that collected fee doesn't show up for me with this branch. It is collected in main
<img width="1498" alt="Screenshot 2023-04-11 at 9 21 12 PM" src="https://user-images.githubusercontent.com/1780212/231218698-4be717b9-0ec0-44f4-893d-2ff51bb7921b.png">

Reply:
<img width="867" alt="Screenshot 2023-04-11 at 9 24 04 PM" src="https://user-images.githubusercontent.com/1780212/231219448-0ab2114b-0eaf-45ef-9eaa-f2440034b12a.png">
Should we update it here as well ? cc @PeerRich I am not sure how long we would remain without fee, so it should reflect that ideally.

Reply:
worth keeping, we will readd it later